bin/component.dart

```diff
@@ -1,21 +1,54 @@
-import 'dart:io';
 import 'package:dev_string_converter/dev_string_converter.dart';
+import "utils.dart";
+
+ConfigFile config = new ConfigFile();
 
 main(List<String> args) async {
-//  await new Directory('${args[0]}').create();
+
   var name = args[0];
-  (await new File('${name}/${name}.dart').create(recursive: true)).writeAsString(
-      '''// Copyright (c) 2016, <your name>. All rights reserved. Use of this source code
+  String lib;
+  String path = "${toTableName(name)}";
+
+  if (config?.componentsPath != null) {
+    path = "${config.componentsPath}/${toTableName(name)}";
+    lib = "${config.componentsPath}/components.dart";
+  }
+
+  String dartPath = '$path/${toTableName(name)}_component.dart';
+  String htmlPath = '$path/${toTableName(name)}_component.html';
+  String cssPath = '$path/${toTableName(name)}_component.css';
+
+  await writeInFile(dartPath, componentTemplateDart(name));
+  await writeInFile(htmlPath, componentTemplateHtml(name));
+  await createFile(cssPath);
+
+  if (lib != null) {
+    addToLibrary("${toTableName(name)}/${toTableName(name)}_component.dart", lib);
+  }
+
+}
+
+String componentTemplateDart(String name) =>
+    '''// Copyright (c) 2016, <your name>. All rights reserved. Use of this source code
 // is governed by a BSD-style license that can be found in the LICENSE file.
 
-import 'package:angular2/angular2.dart';
+import 'package:angular2/core.dart';
 
 @Component(
   selector: '${toPolyName(name)}',
-  templateUrl: '$name.html')
-class ${toUpperCamelCase(name)} {
+  templateUrl: '${toTableName(name)}_component.html',
+  styleUrls: const ['${toTableName(name)}_component.css'])
+class ${toUpperCamelCase(name)} implements OnInit {
+
+  ${toUpperCamelCase(name)}() {}
```

Comment:
I don't think is good idea creating constructors since we don't know if they are going to be needed. Furthermore, dart code-style recomends not using `{}` for empty consructors, instead it should be used `;`